src/components/studio-footer/StudioFooter.jsx

```diff
@@ -32,76 +30,18 @@ const StudioFooter = ({
   containerProps,
 }) => {
   const intl = useIntl();
-  const [isOpen, setIsOpen] = useState(false);
   const { config } = useContext(AppContext);
 
   const { containerClassName, ...restContainerProps } = containerProps || {};
 
   return (
     <>
-      <div className="m-0 mt-6 row align-items-center justify-content-center">
-        <div className="col border-top mr-2" />
-        <Button
-          data-testid="helpToggleButton"
-          variant="outline-primary"
-          onClick={() => setIsOpen(!isOpen)}
-          iconBefore={Help}
-          iconAfter={isOpen ? ExpandLess : ExpandMore}
-          size="sm"
-        >
-          {isOpen ? intl.formatMessage(messages.closeHelpButtonLabel)
-            : intl.formatMessage(messages.openHelpButtonLabel)}
-        </Button>
-        <div className="col border-top ml-2" />
-      </div>
+      <StudioFooterHelpSectionSlot />
```

Comment:
I'd like to verify that this still appears correct since this change leads to multiple `Container`s where there used to only be 1.

Before it was 
```html
<HelpButton />
<Container>
  <HelpContent />
  <MoreContent />
</Container>
```

and with this change it's becoming
```html
<HelpButton />
<Container>
  <HelpContent />
</Container>
<Container>
  <MoreContent />
</Container>
```